recipe/meta.yaml

```diff
@@ -7,26 +7,26 @@ package:
   version: {{ version }}
 
 source:
-  url: https://cran.r-project.org/src/base/R-4/R-{{ version }}.tar.gz
-  sha256: b4cb675deaaeb7299d3b265d218cde43f192951ce5b89b7bb1a5148a36b2d94d
-  patches:
-    - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
-    - 0002-Fix-trio-config.h-include-depth-issue.patch
-    - 0003-Win32-Do-not-link-static-libgcc.patch
-    - 0004-Win32-Prevent-conversion-of-R_ARCH-to-abs-Windows-pa.patch
-    - 0005-Darwin-Avoid-setting-DYLD_FALLBACK_LIBRARY_PATH.patch
-    - 0006-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
-    - 0007-javareconf-Do-not-fail-on-compile-fail.patch
-    - 0008-Revert-part-of-9b818c6dc00143ff18775a4015a3f43b5196f.patch
-    - 0009-javareconf-macOS-Continue-to-allow-system-Java-lt-9-.patch
-    # - 0010-Allow-rpath-in-tcltk-dylib-LC_LOAD_DYLIB-command.patch
-    - 0011-Check-for-changes-then-forcibly-mv-in-javareconf.in.patch
-    - 0012-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
-    - 0013-Try-forcing-.dylib.patch
+  - url: https://cran.r-project.org/src/base/R-4/R-{{ version }}.tar.gz
+    sha256: b4cb675deaaeb7299d3b265d218cde43f192951ce5b89b7bb1a5148a36b2d94d
+    patches:
+      - 0001-Darwin-Remove-unicode-elipsis-character-from-grDevic.patch
+      - 0002-Fix-trio-config.h-include-depth-issue.patch
+      - 0003-Win32-Do-not-link-static-libgcc.patch
+      - 0004-Win32-Prevent-conversion-of-R_ARCH-to-abs-Windows-pa.patch
+      - 0005-Darwin-Avoid-setting-DYLD_FALLBACK_LIBRARY_PATH.patch
+      - 0006-Linux-Do-not-modify-LD_LIBRARY_PATH.patch
+      - 0007-javareconf-Do-not-fail-on-compile-fail.patch
+      - 0008-Revert-part-of-9b818c6dc00143ff18775a4015a3f43b5196f.patch
+      - 0009-javareconf-macOS-Continue-to-allow-system-Java-lt-9-.patch
+      # - 0010-Allow-rpath-in-tcltk-dylib-LC_LOAD_DYLIB-command.patch
+      - 0011-Check-for-changes-then-forcibly-mv-in-javareconf.in.patch
+      - 0012-Use-LAPACK_LDFLAGS-in-Rlapack_la_LIBADD.patch
+      - 0013-Try-forcing-.dylib.patch
+      - 0018-Fix-path-to-TCL-TK.patch
 
 build:
-  skip: true  # [win]
-  number: 0
+  number: 1
```

Comment:
The merge includes the `pcre2`=`10.44` migration via https://github.com/conda-forge/r-base-feedstock/pull/310/commits/5b56e49af0d055ea978b115550424aa68b5a74db which is why this has a version bump.